translations/plurals.ts

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.0" language="en">
```

Comment:
Harmattan tools are old. Every time we touch the file it will be changed to 2.0

Beside, this is the file that is loaded by default in order to get plurals to work. It is not related to the polish translation at all :)